### DISCUSSION TITLE
ga-Z97X-Gaming 3 fan control not work

### DISCUSSION THREAD

**Discussion by NTAmmar** [Rem0o/FanControl.Releases#1262] [Q&A]
the program can detect the fan speed but cannot control, i found the monitoring processor are ITE IT8620E.

**Comment by Rem0o** [comment]
See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility

**Reply by NTAmmar** [reply]
> See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility

![SharedScreenshot](https://user-images.githubusercontent.com/92214498/200595440-6166dffc-f386-4225-8645-9ac1e8737c48.jpg)


i found the iissue, as you can see the INTAKE only Control fan #2 or in my motherboard it control CPU OPT. if i pair to other fan (1 3 4 5) it still control the fan no 2, not the choosen fan (1 3 4 5)
it same as other control, only control the sensor below it

**Reply by Rem0o** [reply]
Pairing doesn’t change what the control actually controls. Pairing is just meant to associate the rpm sensor with its inherent control.  In your case, you got some channels that have no control. So there is no pairing to do with those, they are read only.

**Reply by NTAmmar** [reply]
Oh, i see...
So then how to add control fan?, as you can see there are only 5 control fan for 8 sensor.

**Reply by Rem0o** [reply]
You can't. You got 5 controls with speed, + 3 extra speed sensors as read-only.
